index.html

```diff
@@ -0,0 +1,449 @@
+        <div class="nav-bar">
+          <ul class="nav-list nav-styles">
+            <li>
+              <img
+                src="./icons-images/home.svg"
+                alt="home-icon"
+                class="my-icon"
+              />Home
+            </li>
+            <li>
+              <img
+                src="./icons-images/Profile.svg"
+                alt="profile-icon"
+                class="my-icon"
+              />Profile
+            </li>
+            <li>
+              <img
+                src="./icons-images/message-reply.svg"
+                alt="messages-icon"
+                class="my-icon"
+              />Messages
+            </li>
+            <li>
+              <img
+                src="./icons-images/History.svg"
+                alt="history-icon"
+                class="my-icon"
+              />History
+            </li>
+            <li>
+              <img
+                src="./icons-images/Task.svg"
+                alt="task-icon"
+                class="my-icon"
+              />Tasks
+            </li>
+            <li>
+              <img
+                src="./icons-images/communities.svg"
+                alt="commuinties-icon"
+                class="my-icon"
+              />Communities
+            </li>
+          </ul>
+        </div>
+        <div class="settings-bar">
+          <ul class="settings-list nav-styles">
+            <li>
+              <img
+                src="./icons-images/Settings.svg"
+                alt="settings-icon"
+                class="my-icon"
+              />Settings
+            </li>
+            <li>
+              <img
+                src="./icons-images/support.svg"
+                alt="support-icon"
+                class="my-icon"
+              />Support
+            </li>
+            <li>
+              <img
+                src="./icons-images/Privacy.svg"
+                alt="privacy-icon"
+                class="my-icon"
+              />Privacy
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="header">
+        <div class="top-header">
+          <div class="search-bar">
+            <div class="search-img click">
+              <input
+                type="image"
+                class="search-icon"
+                src="./icons-images/search.svg"
+                alt="search-icon"
+                form="my-form"
+              />
+            </div>
+            <form action="#" id="my-form" method="get">
+              <input
+                type="text"
+                class="search-input"
+                id="search-result"
+                name="search-result"
+                value=""
+              /><br /><br />
+            </form>
+          </div>
+          <div class="profile-bar">
+            <div class="click">
+              <img
+                src="./icons-images/bell-ring.svg"
+                alt="notification-icon"
+                class="notification-icon"
+              />
```

Comment:
### Wow, your design is excellent 👏 👏.  I ❤️  this. 
But I notice a few issues, some of your icons are not displaying from my end and I checked to see why that is the case but found nothing. Kindly check this and fix it.

- The bell-icon on the top-right of the screen.
- The icons in the projects cards.

Your Design
![image](https://user-images.githubusercontent.com/84156714/196051372-e6e0d569-42b1-4ece-b50d-39d565cf0677.png)